float64[] e_ddot_star
